server/graphql/common/user.ts

```diff
@@ -87,6 +87,6 @@ export const hasSeenLatestChangelogEntry = async (user: typeof models.User): Pro
   if (!latestChangelogUpdatePublishDate) {
     return true;
   } else {
-    return user?.changelogViewDate >= latestChangelogUpdatePublishDate;
+    return new Date(user?.changelogViewDate) >= new Date(latestChangelogUpdatePublishDate);
```

Comment:
My understanding is that we will end up with a string here (rather than a date) only when we load stuff from the cache (because it has been serialized). `user.changelogViewDate` is already a date, and `latestChangelogUpdate.publishedAt` too.

We can do something like:

```es6
const dateFromCache = await cache.get(cacheKey);
if (dateFromCache) {
  return user?.changelogViewDate >= new Date(dateFromCache);
} else {
  ...
}
```

Reply:
@Betree : But then the doubt I have is why does it only appear in staging not in our local? 🤔 That's the part that I don't quite get. However, let us try your solution. I have made a fix, let me know if it looks good. 😄